File: rafael/operadores/TestaOperacoesString.cs
class TestaOperacoesString 
{
	static void Main() 
	{
		string s = "Rafael Cosentino";
		
		System.Console.WriteLine(s[7]);
		
		System.Console.WriteLine(s.Contains("Cosentino"));
		System.Console.WriteLine(s.Contains("Hirata"));
		
		System.Console.WriteLine(s.EndsWith("Cosentino"));
		System.Console.WriteLine(s.EndsWith("Hirata"));
		
		System.Console.WriteLine(s.StartsWith("Rafael"));
		System.Console.WriteLine(s.StartsWith("Marcelo"));

		s = s.Replace("Rafael", "Jonas");
		System.Console.WriteLine(s);

		System.Console.WriteLine(s.Substring(6));
		System.Console.WriteLine(s.Substring(0, 5));

		s = s.ToUpper();
		System.Console.WriteLine(s);

		s = s.ToLower();
		System.Console.WriteLine(s);

		s = "     K19 Treinamentos     ";
		System.Console.WriteLine(s.Trim());
	}
}

File: rafael/operadores/TestaOperacoesString.java
class TestaOperacoesString {
	public static void main(String[] args) {
		String s = "Rafael Cosentino";
		
		System.out.println(s.charAt(7));
		
		System.out.println(s.contains("Cosentino"));
		System.out.println(s.contains("Hirata"));
		
		System.out.println(s.endsWith("Cosentino"));
		System.out.println(s.endsWith("Hirata"));
		
		System.out.println(s.startsWith("Rafael"));
		System.out.println(s.startsWith("Marcelo"));

		s = s.replaceAll("Rafael", "Jonas");
		System.out.println(s);

		System.out.println(s.substring(6));
		System.out.println(s.substring(0, 5));

		s = s.toUpperCase();
		System.out.println(s);

		s = s.toLowerCase();
		System.out.println(s);

		s = "     K19 Treinamentos     ";
		System.out.println(s.trim());
	}
}
